plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -694,18 +697,15 @@ private class DokkaDescriptorVisitor(
     }
 
     private fun AnnotationDescriptor.toAnnotation(): Annotations.Annotation? =
```

Comment:
Why returning optional here? We know, that we always return something, right?

Reply:
Previously, this function used to return null. Fixed